### PR TITLE
Feature/countdown component

### PR DESCRIPTION
**Pull request by DaQuirm**
There're now three testing scripts in `package.json` :smiley_cat: 
* `test-frontend`
* `test-backend`
* `test` which runs both of the above

Plus, a little nexus wrapper over setInterval :smiley_cat:

### CODE REVIEW

File: app/common/components/countdown/countdown.coffee [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+		@remaining = new nx.Cell
+			'<-': [@time]
+
+		@active = new nx.Cell
+			'<-': [@time, -> yes]
+			value: no
+			action: (value) =>
+				clearInterval @_interval
+				if value
+					@_interval = setInterval interval_handler, ONE_SECOND
```
Comment (kirilknysh) [inline app/common/components/countdown/countdown.coffee]: Where is `@_interval` cleared?

Comment (DaQuirm) [inline app/common/components/countdown/countdown.coffee]: see line 25

**Comment by AndreyLukashenko** [comment]
LGTM

**Comment by kirilknysh** [comment]
Look OK;

**Pull request merged**